test/e2e/installplan_e2e_test.go

```diff
@@ -3320,15 +3320,6 @@ var _ = Describe("Install Plan", func() {
 	It("compresses installplan step resource manifests to configmap references", func() {
 		// Test ensures that all steps for index-based catalogs are references to configmaps. This avoids the problem
 		// of installplans growing beyond the etcd size limit when manifests are written to the ip status.
-
-		og := &operatorsv1.OperatorGroup{}
```

Comment:
It's not clear why this block was removed?

Reply:
I removed this as [this code](https://github.com/operator-framework/operator-lifecycle-manager/pull/3042/files#diff-eddfb8217b77c532b776defca62eb7d6e4710bb15ceb8bf8a4d8de408df3a6a4R72) will create a fresh namespace with an operator group in it.